legion/tests/test_models/columns_model/run.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+
+import legion.model
+
+
+legion.model.init('columns model', '1.0')
+
+
+def calculate(x):
+    return list(x.keys())
```

Comment:
Please address values by indexes (this is what ML models really do under the hood)

Reply:
I access column names here. Not sure if it is possible to access column names by index.

Or maybe you mean I should rewrite test to compare DataFrame values instead of column names?